src/turnkeyml/cli/report.py

```diff
@@ -78,11 +78,34 @@ def summary_spreadsheets(args) -> None:
 
                         # Copy the build-specific stats
                         for key, value in build.items():
-                            # Break each value in "completed build stages" into its own column
-                            # to make analysis easier
+                            # Break each value in "completed build stages" into status and duration
+                            # to make the analysis of this data easier
                             if key == fs.Keys.COMPLETED_BUILD_STAGES:
-                                for subkey, subvalue in value.items():
-                                    evaluation_stats[subkey] = subvalue
+                                previous_state_incomplete = False
+                                for stage in build[fs.Keys.SELECTED_SEQUENCE_OF_STAGES]:
+                                    duration_column_name = f"stage_duration: {stage}"
+                                    state_column_name = f"stage_status: {stage}"
+                                    if stage in build[fs.Keys.COMPLETED_BUILD_STAGES]:
+                                        evaluation_stats[
+                                            state_column_name
+                                        ] = "COMPLETED"
+                                        evaluation_stats[duration_column_name] = build[
+                                            fs.Keys.COMPLETED_BUILD_STAGES
+                                        ][stage]
+                                    elif not previous_state_incomplete:
+                                        previous_state_incomplete = True
+                                        evaluation_stats[
+                                            state_column_name
+                                        ] = "INCOMPLETE"
```

Comment:
Please unify the terminology between the [build status](https://github.com/onnx/turnkeyml/blob/a52414c1abb46782f2b37c9d8b60cd8b145496e0/src/turnkeyml/common/filesystem.py#L363) and the stage status. They are conceptually similar and so should use similar terms. I don't mind if you want to adjust one or the other, as long as they end up cohesive. 

For example, `COMPLETED` vs. `SUCCESSFUL`: I don't want to have to answer user questions like "what is the difference between a 'completed' status and a 'successful' status?" 😉 

Ideally you use this single Enum to contain all the status values https://github.com/onnx/turnkeyml/blob/a52414c1abb46782f2b37c9d8b60cd8b145496e0/src/turnkeyml/common/filesystem.py#L363

Reply:
Terminology unified. Everything is called "COMPLETED" now.

Reply:
After looking at the changes, I think the term should actually be "SUCCESSFUL" not "COMPLETED". 

Semantically, there is a big difference between the two terms. Primarily, prior to this PR, we base our cache hits on whether the build status is successful or not. Under this change, the cache hit is based on whether the build completed... which does not have the same semantic clarity. A build could be both "completed" and not suitable for a cache hit, but a "successful" build is obviously suitable for a cache hit.

Let me know if you want me to jump in on this...

Reply:
Implemented!